sentry-android-core/src/test/java/io/sentry/android/core/DefaultAndroidEventProcessorTest.kt

```diff
@@ -456,6 +462,25 @@ class DefaultAndroidEventProcessorTest {
         }
     }
 
+    @Test
+    fun `Does not collect device info that requires IPC if disabled`() {
```

Comment:
Would be good to check that by default (no change in SentryOptions) we do get the additional context

Reply:
@bruno-garcia we already had a test for that which does what you're asking right above this test. See https://github.com/getsentry/sentry-java/pull/2100/files#diff-47a15d06d787ba053ff683e5e72e2d105eac60e27127cb8938744247dd4f7593R443-R463